test/cases/grid-template.out.css

```diff
@@ -91,3 +91,34 @@
     -ms-grid-column-span: 9;
     grid-area: ft;
 }
+
+@media (min-width: 1000px) {
+    .b {
+        -ms-grid-rows:
+            30px auto 1fr 30px;
```

Comment:
I think we should not have new line here and l in next property. Let's clean `between` here too

Reply:
It's a little different case. These declarations extends `between` from origin which already had new line. So it's better to control further fixes with user's styleguide or linter. But I can fix test and remove new line from origin.

Reply:
@ai What do you think?

Reply:
@ai About my second commet

Reply:
Just remove `between`. Do not copy it from an origin. PostCSS will fill it automatically with own space detect algorithm.